ydb/core/kqp/ut/olap/indexes_ut.cpp

```diff
@@ -113,8 +113,8 @@ Y_UNIT_TEST_SUITE(KqpOlapIndexes) {
         TKikimrRunner kikimr(settings);
 
         auto csController = NYDBTest::TControllers::RegisterCSControllerGuard<NYDBTest::NColumnShard::TController>();
-        csController->SetOverridePeriodicWakeupActivationPeriod(TDuration::Seconds(1));
-        csController->SetOverrideLagForCompactionBeforeTierings(TDuration::Seconds(1));
+        csController->SetPeriodicWakeupActivationPeriod(TDuration::Seconds(1));
```

Comment:
These should be renamed back after merging https://github.com/ydb-platform/ydb/pull/8000